firebase-config/src/main/java/com/google/firebase/remoteconfig/internal/ConfigFetchHttpClient.java

```diff
@@ -71,9 +77,11 @@
  * @author Lucas Png
  */
 public class ConfigFetchHttpClient {
+
   private static final String API_KEY_HEADER = "X-Goog-Api-Key";
   private static final String ETAG_HEADER = "ETag";
   private static final String IF_NONE_MATCH_HEADER = "If-None-Match";
+  private static final String TRANSPORT_FINAL = "-1"; //(TODO) Replace with actual logSource int
```

Comment:
Is this a header? If so, add the "_HEADER" suffix for consistency. If not, move the declaration down and keep the headers grouped together

Reply:
It is not a header. I'll move it!